requirements.txt

```diff
@@ -0,0 +1 @@
+agpypeline==0.0.20
```

Comment:
This could potentially be changed to the latest version of agpypeline, 0.0.37

Reply:
It hasn't been tested with that version. I will put in a ticket for updating the repos once we have a "final" agpypeline version.